lib/smart_answer_flows/marriage-abroad.rb

```diff
@@ -733,8 +733,8 @@ def define
             end
           end
 
-          if ceremony_country == 'italy' && resident_of != 'uk'
-            phrases << :required_supporting_documents_italy
+          if resident_of != 'uk' && data_query.phrase_exists?("required_supporting_documents_#{ceremony_country}")
```

Comment:
Do you mean something like this: https://github.com/alphagov/smart-answers/blob/efe056e4f6b67efb2677e0e868f0b50b88774a2d/lib/smart_answer_flows/marriage-abroad.rb#L373-L475?

If so it would make it easier to make changes that affect a single country, but it would also introduce some duplication in logic that would make it more difficult to make changes that affect multiple countries. In this flow both things happen quite often.